docs/cli-tool/cli-commands/genezio-create.md

```diff
@@ -10,18 +10,34 @@ description: Discover how to create fullstack or backend projects with the genez
 </head>
 ### Usage
 
-#### Interactive
```

Comment:
Please keep the old indentation (4 `#`), these sections are under `Usage` which also has 3 `#`.